lib/analysis.js

```diff
@@ -305,7 +352,26 @@ function analyseAll() {
             });
             c(node.body, initNodeInfo(nodeInfo, "fn_body"));
         },
+        ExportNamedDeclaration: function(node, nodeInfo, c) {
+            for (var i = 0; i < node.specifiers.length; i++) {
+                var spec = node.specifiers[i];
+            }
+
+            if (node.source) {
+                emitModuleRef(node, node.source);
+            }
+        },
+        ImportDeclaration: function(node, nodeInfo, c) {
+            for (var i = 0; i < node.specifiers.length; ++i) {
+                if (node.specifiers[i].local) {
+                    c(node.specifiers[i].local, initNodeInfo("var"));
```

Comment:
for (let i = 0; i < node.specifiers.length; i++) {
                 ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode